scripts/gas-refund-program/computeMerkleTree.ts

```diff
@@ -56,15 +55,15 @@ export async function computeAndStoreMerkleTreeForChain({
 }
 
 async function startComputingMerkleTreesAllChains() {
+  await init();
```

Comment:
I confirm.

It has to be manual because:
- gives an extra window to verify data
- seed the contract with the root before making it available to users

To achieve this right now there's 2 commands `gas-refund:compute-merkle-tree-save-file` and `gas-refund:compute-merkle-tree-save-db`. Refer to [readme](https://github.com/paraswap/paraswap-volume-tracker/blob/9212ab38230c9b41387ff1345859b86a4a7f7083/scripts/gas-refund-program/readme.md) for full explanation.